apps/site/assets/ts/shuttles/__tests__/ShuttlesPageTest.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import renderer from "react-test-renderer";
+import { mount, ReactWrapper } from "enzyme";
```

Comment:
@thecristen I'd like commit titles to contain enough information that, when you're skimming the logs and looking at just the titles, you have enough context to tell what in the app is changing.

In this case, if I were to read just the title of the commit, I would only know that it was a special case for the Green Line for something-or-another, and there are lots of ways in which the Green Line is an edge case, so the context is not as clear as is desirable.

What I suggest is a commit title like:

`Handle Green Line in shuttle direction selector`

and then in the body of the commit message you can explain the specifics of the fix, i.e. in this case using the generic labels "Eastbound/Westbound".